### DISCUSSION TITLE
Default Font-Family for Text?

### DISCUSSION THREAD

**Discussion by megacherry** [tamagui/tamagui#734] [Q&A]
Hi all,

it took me a little while to integrate custom fonts. Is there a way to add a default font family to the default text component?

Currently, this is the only way it works for me: 

```
import {Text} from '@tamagui/core'

<Text style={{fontFamily: 'Quicksand'}}>foo</Text>
```

Thank you.

**Comment by megacherry** [comment]
If anyone comes across this question: As I understood it, SizableText is the solution :)

**Comment by supto09** [comment]
For any future reader i tried to summarize the steps in medium [Custom Font In React Native](https://medium.com/@ashiksaeedsupto_46134/tamagui-ui-kit-custom-fonts-in-react-native-18b3669c7d29)
